onnxmltools/convert/common/_apply_operation.py

```diff
@@ -438,3 +439,46 @@ def apply_softmax(scope, input_name, output_name, container, operator_name=None,
 def apply_normalization(scope, input_name, output_name, container, operator_name=None, axis=1, p=2):
     name = _create_name_or_use_existing_one(scope, 'LpNormalization', operator_name)
     container.add_node('LpNormalization', input_name, output_name, name=name, p=p, axis=axis)
+
+def apply_crop_height_width(scope, input_name, output_name, container, operator_name=None,
```

Comment:
consistent function naming convention is "apply_crop" here!

Reply:
This function specifically crops H and W in [N, C ,H, W] coordinate system, so `crop` sounds too general in this case.